opencapco/opencap.co
# Transfer landing page from opencap.rubric.sh

Testing:
- Install dependencies: `bun i`
- Copy example env: `cp .env.example .env`
- Run dev server: `bun run dev`
- Open `localhost:3000` in browser
- Saved changes should hot-reload in near-realtime

Sample:

Hero:
<img width="1311" alt="image" src="https://github.com/opencapco/opencap.co/assets/36117635/14b4467a-1cba-4fe8-b96f-79fdacd24293">

Footer:
<img width="1310" alt="image" src="https://github.com/opencapco/opencap.co/assets/36117635/59615f80-8fe0-43c0-999a-01bb44b4db72">

Social preview:
<img width="704" alt="image" src="https://github.com/opencapco/opencap.co/assets/36117635/bd76e3ad-91ec-45f5-a46e-bdc84b2bfbc5">
